doc/user_guide/marks.rst

```diff
@@ -60,7 +60,73 @@ arguments to represent points as red semi-transparent filled circles:
        y='Miles_per_Gallon:Q'
    )
 
-The additional arguments to ``mark_*()`` methods are passed along to an
+Similarly, the compound mark :meth:`~Chart.mark_boxplot` can be used to create a boxplot without having to specify each part of the plot (box, whiskers, outliers) separately.
```

Comment:
I would put all of this in its own section:
```
Compound Marks
--------------

BoxPlot
~~~~~~~
```

Reply:
Good idea, I made the change and also added a section on mark properties (to help the transition).